docs/sources/datasources/postgres/_index.md

```diff
@@ -98,8 +98,8 @@ The response from PostgreSQL can be formatted as either a table or as a time ser
 
 ### Dataset and Table selection
 
-In the dataset dropdown, choose the PostgreSQL database to query. The dropdown is be populated with the databases that the user has access to.
-When the dataset is selected, the table dropdown is populated with the tables that are available.
+The dataset dropdown will be disabled by default, and is populated with the configured database to which the user has access.
```

Comment:
This is a bit confusing.....If the dropdown is disabled, how can you see the databases that you can access? I am not clear on the wording here. Can you clarify?

Reply:
Sure thing! What I mean is, Postgres databases will still have a dataset dropdown, but will only have access to the preconfigured database. So the dropdown will still exist, but it will be disabled, and have a placeholder of the single database available. Does that make sense?

Reply:
yes it does! thank you